docs/high-avaliability/prometheus-HA.md

```diff
@@ -1,12 +1,62 @@
+# Running Promscale itself in High-Availability mode
+
+Promscale is a stateless service, thus it can safely be run with multiple
+replicas. A load-balancer can simply route Promscale requests to any replica.
+
 # Using Promscale with Prometheus in High Availability
 
-Promscale supports running Prometheus in High Availability (HA). This means that more than one Prometheus instances can run in parallel. Promscale implements leader election to support Prometheus HA mode through pg_advisory_lock.
+Promscale supports running Prometheus in High Availability (HA). This means
+that more than you can have multiple Prometheus instances running as "HA
+pairs". Such "HA pairs" (what we'll call clusters from now on) are simply two
+or more identical Prometheus instance running on different
+machines/containers. These instances scrape the same targets and should thus
+have the same data (modulo timing of the scrapes).
```

Comment:
Not sure if `modulo timing of the scrapes` is clear to the users. Instead, a `with same scrape interval` is clearer.